modules/sdk/project-templates/project-templates-mvcbean-cdi-jsp-portlet/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+
+<archetype-descriptor name="liferay-module-mvcbean-cdi-jsp-portlet">
+	<fileSets>
+		<fileSet filtered="true" packaged="true">
+			<directory>src/main/java</directory>
+			<includes>
+				<include>**/*.java</include>
+			</includes>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/resources</directory>
+			<includes>
+				<include>**/*</include>
+			</includes>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/webapp</directory>
+			<excludes>
+				<exclude>*.png</exclude>
+				<exclude>**/*.jspx</exclude>
```

Comment:
@gamerson Right -- the JSPX files can't be filtered because of EL expressions like `${namespace}` getting goofed up by Maven during filtering. No need to filter them anyway -- nothing to substitute.